### PR TITLE
feat: add probes to helm chart

### PR DESCRIPTION
**Pull request by mbathe19**
Hi @Lerentis,

adding the prob-values to the values file.

Regards,

Michel

### CODE REVIEW

**Review by Lerentis — CHANGES_REQUESTED** [review]
Hi @mbathe19 ,

Thanks for the PR. Can you bump the chart version and list the change there as well?

**Comment by mbathe19** [comment]
Hi @Lerentis,
added Changelog and bumped Chart Version.
Hopefully its now ok for you?

File: charts/bitwarden-crd-operator/Chart.yaml [hunk 1]
```diff
@@ -95,8 +95,8 @@ annotations:
   artifacthub.io/operator: "true"
   artifacthub.io/containsSecurityUpdates: "false"
   artifacthub.io/changes: |
-    - kind: fixed
-      description: "Fixed bitwarden installation"
+    - kind: feat
```
Comment (Lerentis) [inline charts/bitwarden-crd-operator/Chart.yaml:98]:
```suggestion
    - kind: added
```
supported kinds are added, changed, deprecated, removed, fixed and security as stated here: https://artifacthub.io/docs/topics/annotations/helm/

**Review by Lerentis — CHANGES_REQUESTED** [review]
just a minor change on the changelog kind and this is good to merge :)

**Review by Lerentis — APPROVED** [review]

**Pull request merged**